cli/pcluster/config/validators.py

```diff
@@ -49,6 +50,10 @@
 
 FSX_SUPPORTED_OSES = ["centos7", "ubuntu1604", "ubuntu1804", "alinux", "alinux2"]
 
+# Constants for section labels
+LABELS_MAX_LENGTH = 64
+LABELS_REGEX = "^[A-Za-z0-9-_]*$"
```

Comment:
are we sure `-` does not need to be escaped? also probably rather than * we want to have + to enforce at least 1 char

Reply:
It seems to work indifferently with or without the escape but I agree it's better escaping it; of course also OK for  ``+`` instead of ``*``